inscriptions/ISic0723.xml

```diff
@@ -169,11 +174,22 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:lang="la">
+                <ab>
+                    <lb n="1"/><expan><abbr>C</abbr><ex>aius</ex></expan> Norb<supplied reason="lost">anus</supplied> <gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost"><expan><abbr>f</abbr><ex>ilius</ex></expan></supplied> <gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost"><expan><abbr>n</abbr><ex>epos</ex></expan> Balbus</supplied>
+                    <lb n="2"/>anno <supplied reason="lost">extremo praeturae</supplied>
+                    <lb n="3"/><supplied reason="lost"><expan><abbr>Q</abbr><ex>uintus</ex></expan> A</supplied>nic<supplied reason="lost" cert="low">ius</supplied> <gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="4"/><supplied reason="lost">quaestor pro <expan><abbr>pr</abbr><ex>aetore</ex></expan></supplied> <gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="5"/><supplied reason="lost">vias in</supplied>cl<supplied reason="lost">inatas et angustas a</supplied>
+                    <lb n="6"/>Syracus<unclear>s</unclear><supplied reason="lost">is ad Acras vorsus</supplied> 
+                    <lb n="7"/>praeter<supplied reason="lost">missis inviis semitis</supplied>
+                    <lb n="8"/>et ab Ac<supplied reason="lost">ris Agrigentum</supplied>
```

Comment:
supplied text at line end should be 'ris ad Agrigentum'

Reply:
Fixed